test/e2e/cypress/e2e/checks_catalog.cy.js

```diff
@@ -93,12 +93,15 @@ context('Checks catalog', () => {
   });
 
   describe('Filtering', () => {
+    const selectedIconSelector =
```

Comment:
question @EMaksy :
why did you create this constant and put in all the array entries?
I mean, you could simply have the constant and use it in the test, right?

Reply:
Very fair point, when working on this first i thought the ui selector would be different, but nope. 
You are right we can just test directly, will create a follow up pr